test/files/neg/t8244.check

```diff
@@ -1,4 +1,4 @@
-Test_2.scala:9: error: value exxx is not a member of ?0
+Test_2.scala:9: error: value exxx is not a member of T
```

Comment:
would be interesting to investigate this one -- should indicate it's an existential skolem